app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,39 @@
+class QueueItemsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @queue_items = current_user.queue_items
+  end
+
+  def create
+    vid = Video.find(params[:video_id])
+    queue_video(vid)
+    redirect_to my_queue_path
+  end
+
+  def destroy
+    queue_item = QueueItem.find(params[:id])
+    queue_item.destroy if current_user_owner?(queue_item)
+    redirect_to my_queue_path
+  end
+
+  private
+
+  #create
```

Comment:
This comment is not helping it. Code is usually read from public methods to private, so private method itself shouldn't really need this comment.